packages/cms/src/member/MetaEditor.jsx

```diff
@@ -745,7 +804,7 @@ class MetaEditor extends Component {
                 {currentRow.imageId && currentRow.dimension && currentRow.id && currentRow.cubeName
                   ? <img
                     className="cms-meta-selected-img"
-                    src={`/api/image?dimension=${currentRow.dimension}&cubeName=${currentRow.cubeName}&id=${currentRow.id}&type=thumb&t=${epoch}`}
+                    src={`/api/image?dimension=${currentRow.dimension}&cubeName=${currentRow.cubeName}&id=${currentRow.id}&size=thumb&t=${epoch}`}
```

Comment:
`type` changed to `size`?

1. I think you missed the `type=splash` in the Hero.jsx component
2. this change should be mentioned in the release notes

Reply:
Nice catch - This is actually a bug, not a change. The imageRoute has always had two query args, `size` and `type`. `size` can be thumb/splash and `type` can only be `json`. 

In the CMS I was *incorrectly* using `type` for `size`, which simply meant that the preview images that were coming through were splash size (unnecessarily). The `type=splash` in the hero component is working by accident basically. 

No docs change needed, just need to invoke my own damn endpoint correctly. change incoming.